### ISSUE TITLE
Add caption and label

### ISSUE THREAD

**Issue by codingyourlife** [olivierverdier/python-latex-highlighting#4]
With lstlisting I can do something like this:
```
\begin{lstlisting}[breaklines,caption={My lstlisting Cpation},captionpos=b,label=lst:lstlistingWithCaptionAndLabel]
#my code for lstlisting that has a caption and label
\end{lstlisting}
```
Not with python-latex-highlighting:
```
\begin{python}[breaklines,caption={DESCRIPTION},captionpos=b,label=lst:LABEL]
#my python-latex-highlighting code where caption and label is ignored
\end{python}
```

Maybe not really a bug, maybe not even a missing feature but I guess this is the best way to contact the author...

**Comment by codingyourlife** [comment]
Okay just found a workaround. Maybe this helps someone else (just add empty lstlisting bellow):
```
\begin{python}
#my python-latex-highlighting with caption bellow because of additional lstlisting
\end{python}
\begin{lstlisting}[frame=none,caption={TEST123},captionpos=b,label=lst:TEST]
\end{lstlisting}
```

**Comment by olivierverdier** [comment]
Well, that's a legitimate request, I'll see if that is possible.

**Comment by jasonzutty** [comment]
+1

**Comment by braingineer** [comment]
not exactly a fix either, but another way of using the package:

```

\begin{lstlisting}[style=mypython,caption={Test}]
def test(x):
    print(x)
\end{lstlisting}
```

**Comment by 1kastner** [comment]
Have you tried to put it into a figure? Is that feasible?

**Comment by haenry** [comment]
I think the best option is to define the python environment in the same way the original lstlistings is defined in _listings.sty_.

I created a PR for this #10

**Comment by CatHas4Paw** [comment]
@braingineer wie deklariert man die neue Variable aus der *.sty datei?

**Comment by samcarter** [comment]
The `python` environment already offers the possibility to add an optional argument, it would just need to also use this argument to facilitate captions, labels etc

```
\lstnewenvironment{python}[1][]{\lstset{style=mypython,#1}}{}
```

**Comment by olivierverdier** [comment]
Fixed in #10
